### ISSUE TITLE
Investigate replacement aircraft for USAF pack

### ISSUE THREAD

**Issue by Fourjays** [blackwatchint/blackwatchint#131]
The USAF mod is large in size and very buggy (C130 damage, bad HUDs, broken GPS targeting, bugged pilot seats, etc). We want to gather a list of alternative mods that can potentially replace the USAF pack. Replacements must be of a high quality, in terms of appearance, functionality and flight characteristics. We would also like the replacements to cover a larger variety of militaries (Russian, British, German, etc) so as to better facilitate the wide variety of factions that we play as.

**Comment by Kentersnoodlefactory** [comment]
Ok. So, i've found a tested a few mods, i will try to get a few more aswell for Germany and the British, will get back on that the next couple of days.

C130 pack: http://www.armaholic.com/page.php?id=28024

advantages over USAF: They fly alot better, they do look alot better with a much nicer HUD and cockpit in general. And they have many diffrent variants

Disadvantage: The cargo variant does not take cargo, afaik you can not load a vehicle in it and drop it. But how often is it used anyways? Do we rather have a more or less non functional airplane with a few extra features or a well working airplane which is actually flyable. 

The one we have got now can't even take of on a shorter runway than 2000+ft which is ridiculous for a c130, if you pull up more than 5 degrees it hits your tail on the ground and can result in a explosion.



 F16C Fighting Falcon: http://www.armaholic.com/page.php?id=27766

Advantages, it looks very nice alot better than the current one we got, it flies better and can actually go faster than 500km/h.

Disadvantages: it does not have the GPS target system, (not that it worked well anyways) And if you land a bit hard it can take damage, you got to remember it is a fragile f16 after all. But performing normal landings is not a problem.



SU35S Flanker E: http://www.armaholic.com/page.php?id=24024

This is a very nice and detailed aircraft, it got a pretty nice cockpit. And it flies very nice, it models the movements of the thrust vectoring. It does fly like a airplane with that feature. It can be a bit sensitive in pitch but in general it is a nice aircraft and i think it would make a nice addition to the Russian hangar.



More coming next few days.........

**Comment by Fourjays** [comment]
Worth mentioning that we already have the F14 and F18 in the modpack for carrier operations, and an SU35 (#134), SU34 (#108) and Antonov AN-2 (#97) have already been suggested. I've found a couple of extra aircraft that should be considered in replacing the USAF pack:

AV-8B Harrier - http://www.armaholic.com/page.php?id=26840
Good quality fighter jet that'd provide another non-US jet (although there are USMC variants included) as well as giving us a VTOL capable jet.

T6 Texan II - http://www.armaholic.com/page.php?id=26067
A single prop training/light attack aircraft. Comes with variants for many factions (BLUFOR, OPFOR and IND) and would give us an aircraft capable of taking off on the shortest runways.

I also looked at an A10 and an SU25 mod, but RHS already includes both aircraft and I couldn't see any discernible differences between the standalone aircraft and the RHS versions.
